src/app/doubtfire-angularjs.module.ts

```diff
@@ -247,6 +246,7 @@ import 'build/src/i18n/resources-locale_en-AU.js';
 import 'build/src/i18n/resources-locale_en-GB.js';
 //#endregion
 
+import { CreateUnitModalComponent } from './admin/modals/create-unit-modal/create-unit-modal.component';
```

Comment:
This line can be removed since it isn't being used in the file

Reply:
Thanks I just made the change